src/Std/Tactic/BVDecide/LRAT/Internal/Formula/RupAddResult.lean

```diff
@@ -1120,7 +1120,7 @@ theorem nodup_derivedLits {n : Nat} (f : DefaultFormula n)
   have li_in_derivedLits : li ∈ derivedLits := by
     have derivedLits_rw : derivedLits = (Array.mk derivedLits).data := by simp only
     simp only [derivedLits_arr_def, li]
-    conv => rhs; rw [derivedLits_rw]
+    conv => rhs; simp (config := {singlePass := true}) only [derivedLits_rw]
```

Comment:
@hargoniX, could you review this? This was the first hacky thing that made it work again. I didn't look to see what is going on.